### PR TITLE
Make robottelo more PEP8 compatible

### PR DESCRIPTION
**Pull request by elyezer**
New version of flake8 started complaining about some methods names. Add
noqa to avoid that on classes which override unittest methods.

Also avoid checking for python version when importing unittest in favor
of using try/except for that. In addition, avoid this kind of logic as
much as possible by using stubbed decorator on test skips.

Finally, fix other minor issues like: first attr of a class method
should be named cls, changing cases during import, indentation of
closing `)]}`

### CODE REVIEW

File: tests/foreman/cli/test_partitiontable.py [hunk 1]
```diff
@@ -22,9 +22,6 @@ def setUp(self):
         self.args = {'name': self.name,
                      'content': self.content}
 
-    def tearDown(self):
-        """Remove the file"""
-
```
Comment (Ichimonji10) [inline tests/foreman/cli/test_partitiontable.py]: :+1:

File: tests/foreman/ui/test_openscap.py [hunk 2]
```diff
@@ -181,7 +181,7 @@ def test_assign_policies_for_hosts(self):
 
         """
 
-    def test_deashboard_Views(self):
+    def test_deashboard_views(self):
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_openscap.py]: `s/deashboard/dashboard/`?

**Comment by Ichimonji10** [comment]
./scripts/graph_entities.py:21:1: E402 module level import not at top of file
    ./scripts/graph_entities.py:22:1: E402 module level import not at top of file
    ./scripts/generate_entities.py:17:1: E402 module level import not at top of file
    ./scripts/generate_entities.py:18:1: E402 module level import not at top of file
    ./docs/conf.py:97:1: E402 module level import not at top of file
    ./docs/conf.py:98:1: E402 module level import not at top of file

File: tests/foreman/ui/test_openscap.py (hunk 2 above)
Comment (elyezer) [inline tests/foreman/ui/test_openscap.py]: Will change this also

**Comment by Ichimonji10** [comment]
ACK

**Comment by JacobCallahan** [comment]
ACK

**Pull request merged**